packages/@lwc/ssr-compiler/package.json

```diff
@@ -0,0 +1,59 @@
+{
+    "//": [
+        "THIS FILE IS AUTOGENERATED. If you modify it, it will be rewritten by check-and-rewrite-package-json.js",
+        "You can safely modify dependencies, devDependencies, keywords, etc., but other props will be overwritten."
+    ],
+    "name": "@lwc/ssr-compiler",
+    "version": "6.1.0",
+    "description": "Compile component for use during server-side rendering",
+    "keywords": [
+        "lwc"
+    ],
+    "homepage": "https://lwc.dev",
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/salesforce/lwc.git",
+        "directory": "packages/@lwc/ssr-compiler"
+    },
+    "bugs": {
+        "url": "https://github.com/salesforce/lwc/issues"
+    },
+    "license": "MIT",
+    "publishConfig": {
+        "access": "public"
+    },
+    "main": "dist/index.cjs.js",
+    "module": "dist/index.js",
+    "types": "dist/index.d.ts",
+    "files": [
+        "dist"
+    ],
+    "scripts": {
+        "build": "rollup --config ../../../scripts/rollup/rollup.config.js",
+        "dev": "rollup  --config ../../../scripts/rollup/rollup.config.js --watch --no-watch.clearScreen",
+        "test-manual": "jest ./src/**/*.spec.ts"
+    },
+    "nx": {
+        "targets": {
+            "build": {
+                "outputs": [
+                    "{projectRoot}/dist"
+                ]
+            }
+        }
+    },
+    "dependencies": {
+        "@lwc/metadata": "6.1.0-0",
+        "@lwc/sfdc-compiler-utils": "6.1.0-0",
+        "@lwc/style-compiler": "6.1.0",
+        "@lwc/template-compiler": "6.1.0",
+        "acorn": "~8.10.0",
+        "astring": "^1.8.6",
+        "estree-toolkit": "^1.7.3",
+        "immer": "^10.0.3",
+        "meriyah": "^4.3.8"
```

Comment:
Curious about the usage of `meriyah` as it doesn't support TS.

Reply:
`@lwc/compiler` also doesn't support TypeScript. It needs to be transpiled (or at least type information needs to be stripped) prior to invocation of the LWC compiler.